shared/engine.js

```diff
@@ -61,7 +62,7 @@ const updateEngine = async ({ status, name, id, alias }) => {
 		});
 		log.updateSuccess('Testing completed.');
 
-		log.success(`${name} v${version} has been installed! 🎉`);
+		log.success(`${name} ${versionToDisplay} has been installed! 🎉`);
```

Comment:
Are the changes in this file necessary?

Reply:
It will print "quickjs vv0.3.0 has been installed!" rather than "quickjs v0.3.0 has been installed!" without this change.  Not a big problem.

Reply:
Thanks. I fixed it in a different way here: 7f5c1563fa38a57b524d0232755d491218482fb5